src/server/tablet_snap.rs

```diff
@@ -314,6 +314,9 @@ async fn cleanup_cache(
             }
         }
         fs::remove_file(entry.path())?;
+        if let Some(m) = key_manager {
+            m.delete_file(entry.path().to_str().unwrap())?;
+        }
```

Comment:
Is there any `EncryptedFilesystem`? Maybe we can wrap these two remove-file in `EncryptedFilesystem::remove_file`?

Reply:
Maybe in another PR, file_system is a drop-in replacement for std::fs, but `remove_encrypted_file` requires additional parameter, and it can't be placed in file_system crate due to cyclic dependency.